### DISCUSSION TITLE
Model

### DISCUSSION THREAD

**Discussion by patschue** [i4Ds/FlareSense#2] [Ideas]
Supervised
- CNN
- RNN/LSTM

[Unsupervised](https://scikit-learn.org/stable/modules/clustering.html)
- K-Means
- DBSCAN
- HDBSCAN

**Comment by gabrieltorresgamez** [comment]
Eine weitere Idee wäre, Noise Reduction auf der Frequenzachse durchzuführen, alle Frequenzen aufeinandersummieren und anhand von Peak Detection Algorithmen Eruptionen zu erkennen. Diese könnte man danach auch je nach Signal/Noise Ratio in die entsprechenden Kategorien A, B, C, M oder X ordnen.
